bmc-discovery/info.json

```diff
@@ -0,0 +1,121 @@
+          "value": 0
+        },
+        {
+          "title": "Results ID",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "results_id",
+          "tooltip": "(Optional) An opaque id that can be used to access a result set that has already been generated via a previous search request. Must be supplied if requesting an offset greater than 0.\nNote that even if retrieving results from an existing results set, the original query must be supplied in case the result set has timed out and the query needs to be re-run.\nThe results_id can be found in any page of results apart from the last one.",
+          "description": "(Optional) An opaque id that can be used to access a result set that has already been generated via a previous search request. Must be supplied if requesting an offset greater than 0.\nNote that even if retrieving results from an existing results set, the original query must be supplied in case the result set has timed out and the query needs to be re-run.\nThe results_id can be found in any page of results apart from the last one."
+        },
+        {
+          "title": "Limit",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "integer",
+          "name": "limit",
+          "tooltip": "(Optional) Specify the maximum number of records that you want this operation to fetch from BMC Discovery.",
+          "description": "(Optional) Specify the maximum number of records that you want this operation to fetch from BMC Discovery.",
+          "value": 100
+        },
+        {
+          "title": "Result Deleted",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "checkbox",
+          "name": "delete",
+          "tooltip": "(Optional) Controls whether the result set is deleted or not before results are returned.\nResult sets are deleted automatically before the last page of results are returned, but setting this field to true on any page will delete the result set immediately.\nThis is an optimization to free resources if you don't plan on accessing any further pages of data. Alternatively, if set to false, automatic deletion of the result set will not happen (for this request).",
+          "description": "(Optional) Controls whether the result set is deleted or not before results are returned.\nResult sets are deleted automatically before the last page of results are returned, but setting this field to true on any page will delete the result set immediately.\nThis is an optimization to free resources if you don't plan on accessing any further pages of data. Alternatively, if set to false, automatic deletion of the result set will not happen (for this request).",
+          "value": false
+        }
+      ],
+      "output_schema": {}
```

Comment:
Output schema is missing.

Reply:
We don't have proper response in API doc that's why we keep as empty.